### PR TITLE
feat/sc 3 database setup

### PR DESCRIPTION
**Pull request by harshitgkm**
- Connect postgres using sequilize

### CODE REVIEW

File: .envsample [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#PORT
+PORT=
+
+#DATABASE VARIABLE
+
+USERNAME=
+PASSWORD=
+DATABASE=
+HOST=
```
Comment (paras2103) [inline .envsample]: variable names are diff from what is used

File: src/config/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const dotenv = require('dotenv');
+dotenv.config();
+//require('dotenv').config();
+
+//console.log("DB Password:", typeof process.env.DB_PASSWORD);
```
Comment (paras2103) [inline src/config/config.js]: remove

File: src/config/config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const dotenv = require('dotenv');
+dotenv.config();
+//require('dotenv').config();
```
Comment (paras2103) [inline src/config/config.js]: remove

**Comment by paras2103** [comment]
after doing the changes, approved

File: .envsample (hunk 1 above)
Comment (harshitgkm) [reply .envsample]: Updated

File: src/config/config.js (hunk 2 above)
Comment (harshitgkm) [reply src/config/config.js]: done

File: src/config/config.js (hunk 3 above)
Comment (harshitgkm) [reply src/config/config.js]: done

**Pull request merged**